### DISCUSSION TITLE
Feature Request: Add keyboard shortcut support for comments.

### DISCUSSION THREAD

**Discussion by daltonfury42** [ananthakumaran/paisa#183] [Ideas]
I see [here](https://codemirror.net/docs/ref/#commands.CommentTokens) that CodeMirror has support for defining and toggling comments. I frequently have to comment out multiple ledger lines after importing the same transaction from two different sources (Zerodha statement and Bank Statement both has the same transaction reported, for example).

If keyboard shortcut to toggle line comment makes sense, can I send a PR for the same on `Ctrl+/`?

**Comment by ananthakumaran** [comment]
I will look at it.
